site/en/docs/lighthouse/accessibility/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: 'layouts/project-landing.njk'
+title: 'Accessibility Audits'
+description: 'These checks highlight opportunities to improve the accessibility of your web app. Only a subset of accessibility issues can be automatically detected so manual testing is also encouraged.'
```

Comment:
since the TOC is already generated per locale, it would be nice if this could use `i18n.docs.lighthouse-accessibility.overview` so you wouldn't need an `index.md` per locale